README.md

```diff
@@ -50,3 +50,9 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+##Configuring Trello API
+First you need to create an account https://trello.com/signup
+Then you need to generate API key and token (explained here https://trello.com/app-key)
+Add them to .env file
```

Comment:
Minor, but it's nice to add the relevant key names here, and also in the `.env.template` file so that it's a little more obvious what's expected

Reply:
Thank you for the suggestion, I have updated it with commit 
f336b8d